baselines/ippo/config.py

```diff
@@ -65,15 +66,15 @@ class ExperimentConfig:
     n_steps: int = 91
     num_minibatches: int = 5  # Used to determine the minibatch size
     verbose: int = 0
-    total_timesteps: int = 1e7
+    total_timesteps: int = 2e7
     ent_coef: float = 0.00
     vf_coef: float = 0.5
     lr: float = 3e-4
     n_epochs: int = 5
 
     # NETWORK
-    mlp_class = LateFusionNet
-    policy = LateFusionPolicy
+    mlp_class = FFN #LateFusionNet
+    policy = FeedForwardPolicy #LateFusionPolicy
```

Comment:
Why default folks to a worse network?

Reply:
The other net assumes the obs will be broken down into ego, visible state etc.. which won't necessarily be the case with LiDAR.

With this net we can still get to 90-100% with lidar only

Reply:
Hmmm, okay, we should revisit this at some point. I don't like outputting a worse net as the default